website/addons/twofactor/static/twoFactorUserConfig.js

```diff
@@ -36,7 +36,9 @@ function ViewModel(qrCodeSelector, otpURL) {
         osfHelpers.postJSON(
             SETTINGS_URL,
             {code: self.tfaCode()}
-        ).done(function() {
+        ).done(function(response) {
+            $('#Tfa-success-message').html(response['message']);
+            $('.close').click();
```

Comment:
This will close any bootstrap-compliant modals, dialogs, or messages - it's too broad for this application.

We only want to close the one message associated with TFA. If we could attach an additional, unique class to that alert message, we could select it specifically...